fp_app/app/models/schedule.rb

```diff
@@ -0,0 +1,25 @@
+class Schedule < ApplicationRecord
+    belongs_to :planner
+    validates :planner_id, presence: true
+    validate :check_schedule
+    validate :check_started_at
+
+    def check_schedule
```

Comment:
メソッドの処理内容に比べてメソッド名の持つ意味が広そうな感じがしました。
もう少し具体的なメソッド名にしても良いかなと思います。
例えば以下
```suggestion
    def check_started_at_future_or_present
```

Reply:
メソッドの命名に悩んだらChat-GPTとかに聞いてみるのもアリですね
MF社内で用意してあるもの↓もあるので使ってみると良いと思います！(Global Protect必要）
https://mf-gen-app.prod.musubu.co.in/

Reply:
ありがとうございます！
メソッド名はわかりやすいように修正しておきます。